styleguide/source/_patterns/01-atoms/table.twig

```diff
@@ -1,42 +1,21 @@
 <table>
-  <caption>How I chose to spend my money</caption>
+  <caption>{{ table.caption }}</caption>
   <thead>
-  <tr>
-    <th data-title="Purchase">Purchase</th>
-    <th data-title="Location">Location</th>
-    <th data-title="Date">Date</th>
-    <th data-title="Evaluation">Evaluation</th>
-    <th data-title="Cost (€)">Cost (€)</th>
-  </tr>
+    <tr>
+      {% for thead in table.theads %}
+        <th data-title="{{ thead.th }}">{{ thead.th }}</th>
+      {% endfor %}
+    </tr>
   </thead>
   <tbody>
-  <tr>
-    <td>Haircut</td>
-    <td>Hairdresser</td>
-    <td>12/09</td>
-    <td>Great idea</td>
-    <td>30</td>
-  </tr>
-  <tr>
-    <td>Lasagna</td>
-    <td>Restaurant</td>
-    <td>12/09</td>
-    <td>Regrets</td>
-    <td>18</td>
-  </tr>
-  <tr>
-    <td>Shoes</td>
-    <td>Shoeshop</td>
-    <td>13/09</td>
-    <td>Big regrets</td>
-    <td>65</td>
-  </tr>
-  <tr>
-    <td>Toothpaste</td>
-    <td>Supermarket</td>
-    <td>13/09</td>
-    <td>Good</td>
-    <td>5</td>
-  </tr>
+  {% for tbody in table.tbodys %}
```

Comment:
I think tables can only one tbody. 
This limits the table to only the number of tds written here
I would loop the tds elements like so:

{% for td in table.tds %}
 <td>{{ td.data }}</td>
{% endfor %}

in your json have an array for td data

Reply:
@designerneil -- yeah, this is exactly why I think we should just build an HTML table using native `<tr><td>` instead of loops. This is wasting our time.